### PR TITLE
feat(rms): add data source to get the list of RMS resource tags

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
 add data source to get the list of RMS resource tags that are in specified project
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1.  add data source to get the list of RMS resource tags that are in specified project
```

## PR Checklist

* [x] Tests added/passed.

```
 make testacc TEST="./huaweicloud/services/acceptance/rms" TESTARGS="-run TestAccDataSourceRmsProjectTags_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/rms -v -run TestAccDataSourceRmsProjectTags_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceRmsProjectTags_basic
=== PAUSE TestAccDataSourceRmsProjectTags_basic
=== CONT  TestAccDataSourceRmsProjectTags_basic
--- PASS: TestAccDataSourceRmsProjectTags_basic (55.83s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/rms       55.904s

```

* [x] Documentation updated.
* [x] Schema updated.
* [ ] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
    \>>>>>> Paste the screenshot here <<<<<<

### CODE REVIEW

File: docs/data-sources/rms_resource_type_tags.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+# huaweicloud_rms_resource_type_tags
+
+Use this data source to get the list of RMS resource type tags.
+
+## Example Usage
+
+```hcl
+variable resource_type {}
```
Comment (saf3dfsa) [inline docs/data-sources/rms_resource_type_tags.md]:
```suggestion
variable "resource_type" {}
```

File: docs/data-sources/rms_resource_type_tags.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+subcategory: "Config"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_rms_resource_type_tags"
```
Comment (houpeng80) [inline docs/data-sources/rms_resource_type_tags.md]:
```suggestion
page_title: "HuaweiCloud: huaweicloud_rms_project_tags"
```
change the resource to this name?

**Pull request merged**